### ISSUE TITLE
Document functions at all

### ISSUE THREAD

**Issue by sourcerer-mike** [FriendsOfPHP/Sami#187]
Sami does not manage to get this documentation:

```
if ( ! function_exists( 'get_term_meta' ) ) {
	/**
	 * Retrieve term meta field for a term.
	 *
	 * @since 1.1.0
	 *
	 * @param int    $term_id  Term ID.
	 * @param string $meta_key The meta key to retrieve.
	 *                         By default, returns data for all keys.
	 * @param null   $single   Whether to return a single value. Default false.
	 *
	 * @return mixed Will be an array if $single is false.
	 *               Will be value of meta data field if $single is true.
	 */
	function get_term_meta( $term_id, $meta_key = '', $single = null ) {
		return get_metadata( 'term', $term_id, $meta_key, $single );
	}
}
```

Is it my syntax or why doesn't it show up in the documentation?

Here is my config: https://github.com/sourcerer-mike/wp-comfort/blob/feature/term-meta/tests/sami.php

**Comment by aik099** [comment]
Does Sami even handle globally defined functions?

**Comment by sourcerer-mike** [comment]
If this is the case, then I change the feature request to "Handle functions in any scope".
I really hope they do.

**Comment by johnbillion** [comment]
Sami doesn't support global procedural functions, unfortunately. See #49.

**Comment by aik099** [comment]
This can be closed because, #49 issue is about the same topic and is closed as well.
